lms/static/scripts/frontend_apps/components/StudentSelector.js

```diff
@@ -4,9 +4,23 @@ import propTypes from 'prop-types';
 import SvgIcon from './SvgIcon';
 
 /**
- * A student navigation tool which shows a student selection list and a previous and next button.
+ * @typedef Student
+ * @prop {string} displayName
+ */
+
+/**
+ * @typedef StudentSelectorProps
+ * @prop {(index: number) => any} onSelectStudent -
```

Comment:
repeat nit comment about the mixed use of full stops and not here.

Reply:
Fixed!